lib/elixir/lib/process.ex

```diff
@@ -239,10 +239,18 @@ defmodule Process do
   which is not alive or when the given PID exits. Note that timers will not be
   automatically canceled when `dest` is an atom (as the atom resolution is done
   on delivery).
-  """
-  @spec send_after(pid | atom, term, non_neg_integer) :: reference
-  def send_after(dest, msg, time) do
-    :erlang.send_after(time, dest, msg)
+
+  The function accepts one boolean option: `:abs`. When `false` (the default),
+  `time` is treated as relative to the current monotonic time. When `true`
+  `time` is an absolute value of the erlang monotonic time when `msg` should
+  be delivered to `dest`. The concept of the erlang monotonic time (and other time
+  measures kept by the VM) can be explored in the
+  [erlang documentation](http://erlang.org/doc/apps/erts/time_correction.html#Erlang_Monotonic_Time).
+  """
+  @spec send_after(pid | atom, term, non_neg_integer, [option]) :: reference
+        when option: {:abs, boolean()}
```

Comment:
I prefer the verbosity of erlang type specs, since it's usually all you need for a quick reference - usually there's no need to go through the docs when you forgot the name of the option.